frontends/web/src/utils/request.ts

```diff
@@ -88,7 +102,10 @@ export function apiGet(endpoint) {
   }).then(response => response.json()).then(handleError(endpoint));
 }
 
-export function apiPost(endpoint, body) {
+export function apiPost(
+  endpoint: string,
+  body?: object | number | string | boolean,
```

Comment:
body should be `any`, right? As JSON.Stringify() works for any type. In the future maybe a post endpoint accepts an array etc.

Reply:
JSON.stringify should throw a TypeError when

- value contains a circular reference
- or [BigInt](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt) value is encountered.


```js
JSON.stringify(BigInt("9007199254740991"))
```

also 

> Only [enumerable own properties](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Enumerability_and_ownership_of_properties) are visited. This means [Map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map), [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set), etc. will become "{}".